hack/test-cmd.sh

```diff
@@ -98,6 +91,53 @@ export OPENSHIFT_PROFILE="${CLI_PROFILE-}"
 # Begin tests
 #
 
+# test client not configured
+[ "$(osc get services 2>&1 | grep 'OpenShift is not configured')" ]
```

Comment:
@smarterclayton  No. 
"Not configured" is a condition that only happens if the client can't guess the server URL from *any* of the expected configuration providers. Users could at this point either provide the `--server` or  `--config=/path/to/config` flags, `KUBECONFIG` or `OPENSHIFTCONFIG`, or config files in the local/home directory locations as expected. That's because we *don't default* to https://localhost:8443 anymore, instead we display this message.
The *only* exception to this rule is `osc login`, which *will prompt* for user input to provide a server, username and password (if not already provided through flags).
See this example:

```
$ osc get services
OpenShift is not configured. You need to run the login command in order to create a default config for your server and credentials:

  osc login

You can also run this command again providing the path to a config file directly, either through the --config flag of the OPENSHIFTCONFIG environment variable.

$ osc login
OpenShift is not configured. Please provide the server URL or hit <enter> to use 'https://localhost:8443': _
```